src/OrchardCore/OrchardCore.Users.Core/Services/UserService.cs

```diff
@@ -69,6 +69,13 @@ public async Task<IUser> AuthenticateAsync(string userName, string password, Act
                 return null;
             }
 
+            if (!(user as User).IsEnabled)
+            {
+                reportError(string.Empty, S["The specified user is not allowed to sign in."]);
```

Comment:
Can you check the `AccountController` @hishamco 

There are two checks there, one for email should be confirmed, the other for account disabled. And the strings are different for both.

Ideally the behaviour should be the same

Reply:
Do you mean this one https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs#L166 if it is that the previous message before last commit looks similar, but Seb suggested that in https://github.com/OrchardCMS/OrchardCore/pull/6637#discussion_r453959838